tests/cnf/ran/diskencryption/diskencryption_suite_test.go

```diff
@@ -0,0 +1,29 @@
+package diskencryption
+
+import (
+	"runtime"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	_ "github.com/openshift-kni/eco-gotests/tests/cnf/ran/diskencryption/tests"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/diskencryption/tsparams"
+	. "github.com/openshift-kni/eco-gotests/tests/cnf/ran/internal/raninittools"
+)
+
+var _, currentFile, _, _ = runtime.Caller(0)
+
+func TestTPM2(t *testing.T) {
+	_, reporterConfig := GinkgoConfiguration()
+	reporterConfig.JUnitReport = RANConfig.GetJunitReportPath(currentFile)
+
+	RegisterFailHandler(Fail)
+
+	RunSpecs(t, "RAN tpm2 tests", Label(tsparams.Labels...), reporterConfig)
+}
+
+var _ = BeforeSuite(func() {
```

Comment:
You mentioned needing a macineconfig patch to allow text mode, could this be checked in the BeforeSuite?

Reply:
fixed